src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
I think it's a good compromise. I have tried to add it in the past but was rejected: #1877